src/gui/sharelinkwidget.h

```diff
@@ -146,6 +156,14 @@ private slots:
     QAction *_unshareLinkAction;
     QAction *_addAnotherLinkAction;
     QAction *_noteLinkAction;
+    QHBoxLayout *_shareLinkLayout;
```

Comment:
Why not use default initialiser for all of them? Like `QHBoxLayout *_shareLinkLayout{}` then you don't need to do it in the constructor. I would also tend to say that this would be more readable.